package.json

```diff
@@ -30,16 +30,16 @@
     "ionicframework"
   ],
   "dependencies": {
-    "@schematics/angular": "^7.0.3",
+    "@angular-devkit/architect": "~0.800.0",
```

Comment:
I don't think these should be dependencies.

Reply:
These are used in the new builders

Reply:
These are used in the new builders